File: backend/scala/src/main/scala/controllers/TestController.scala
package controllers

import org.springframework.web.bind.annotation.{GetMapping, PathVariable, RequestMapping, RestController}

@RestController
@RequestMapping(Array("/customers"))
class TestController {
  @GetMapping(value = Array("/{id}"),
    produces = Array("application/json"))
  def getCustomer(@PathVariable("id") id: Long): Long = id

}

File: backend/java/src/main/java/controllers/TestController.java
package controllers;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class TestController extends ControllerBase {
    /**
     * Toggle play/pause
     */
    @RequestMapping(value = "/test")
    public String test() {
        return "sd";
    }
}
